lib/Models/Terria.ts

```diff
@@ -882,6 +882,33 @@ export default class Terria {
         setViewerMode(viewerMode, this.mainViewer);
       }
     }
+    const useNativeResolution = <boolean>(
+      this.getLocalProperty("useNativeResolution")
+    );
+    if (isDefined(useNativeResolution) && useNativeResolution !== null) {
+      this.setUseNativeResolution(useNativeResolution);
+    }
+
+    const baseMaximumScreenSpaceError = parseFloat(
+      this.getLocalProperty("baseMaximumScreenSpaceError")?.toString() || ""
+    );
+    if (
+      isDefined(baseMaximumScreenSpaceError) &&
+      baseMaximumScreenSpaceError !== null &&
+      baseMaximumScreenSpaceError !== NaN
```

Comment:
This comparison will not work (`NaN` cannot be used in comparisons, e.g. `NaN !== NaN` is `true`). I think `!isNaN(baseMaximumScreenSpaceError)` is sufficient (since `parseFloat` must return `NaN` or a valid number).